### PR TITLE
documentation for new component bmw_connected_drive

### PR DESCRIPTION
**Pull request by ChristianKuehnel**
**Description:**
this is the inital documentation for the new component `bmw_connected_drive`

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12277

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/bmw_connected_drive.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: "BMW connected drive"
+description: "Instructions on how to setup your BMW connected drive account with Home Assistant."
+date: 2018-01-10 23:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.65
```
Comment (MartinHjelmare) [inline source/_components/bmw_connected_drive.markdown]: 0.64

File: source/_components/bmw_connected_drive.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+This component lets you retrieve data on your BMW vehicle from the BMW Connected Drive portal. You need to have a working BMW Connected Drive account and a Connected Drive enabled vehicle for this to work.
+
+For compatibility with your BMW vehicle check the (bimmer_connected page)[https://github.com/ChristianKuehnel/bimmer_connected] on github.
+
+To enable this component in your installation, add the following to your
+`configuration.yaml` file:
+
+{% raw %}
```
Comment (MartinHjelmare) [inline source/_components/bmw_connected_drive.markdown]: I think you only need raw tags around templates. Someone can correct me if I'm wrong.

File: source/_components/bmw_connected_drive.markdown [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    username: #your user name
+    password: #your_password>
+    country: #country of your Connected Drive account
+```
+{% endraw %}
+
+
+{% configuration %}
+bmw_connected_drive:
+    description: configuration
```
Comment (MartinHjelmare) [inline source/_components/bmw_connected_drive.markdown]: This is using four space indent. I think 2 spaces is enough.

File: source/_components/bmw_connected_drive.markdown [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    required: true
+    type: map
+    keys:
```
Comment (MartinHjelmare) [inline source/_components/bmw_connected_drive.markdown]: This config layout is more complex than needed. You only need to add the keys under the component, not the component itself.

File: source/_components/device_tracker.bmw_connected_drive.markdown [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+title: "BMW connected drive"
+description: "Instructions on how to setup your BMW connected drive account with Home Assistant."
+date: 2018-01-10 23:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.65
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.bmw_connected_drive.markdown]: 0.64

File: source/_components/sensor.bmw_connected_drive.markdown [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+title: "BMW connected drive"
+description: "Instructions on how to setup your BMW connected drive account with Home Assistant."
+date: 2018-01-10 23:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Other
+ha_release: 0.65
```
Comment (MartinHjelmare) [inline source/_components/sensor.bmw_connected_drive.markdown]: 0.64

File: source/_components/device_tracker.bmw_connected_drive.markdown (hunk 5 above)
Comment (ChristianKuehnel) [reply source/_components/device_tracker.bmw_connected_drive.markdown]: fixed

File: source/_components/sensor.bmw_connected_drive.markdown (hunk 6 above)
Comment (ChristianKuehnel) [reply source/_components/sensor.bmw_connected_drive.markdown]: fixed

File: source/_components/bmw_connected_drive.markdown (hunk 1 above)
Comment (ChristianKuehnel) [reply source/_components/bmw_connected_drive.markdown]: fixed

File: source/_components/bmw_connected_drive.markdown (hunk 2 above)
Comment (ChristianKuehnel) [reply source/_components/bmw_connected_drive.markdown]: ok, removed them

File: source/_components/bmw_connected_drive.markdown (hunk 3 above)
Comment (ChristianKuehnel) [reply source/_components/bmw_connected_drive.markdown]: fixed

File: source/_components/bmw_connected_drive.markdown (hunk 4 above)
Comment (ChristianKuehnel) [reply source/_components/bmw_connected_drive.markdown]: fixed

**Pull request merged**